### PR TITLE
PSD-854 Remove ES lint and upgrade to node v12 in Acceptance Tests

### PR DESCRIPTION
**Pull request by spattisson**
Remove ES lint and upgrade to node v12 in Acceptance Tests
https://eaflood.atlassian.net/browse/PSD-854
General tidy up of some tech debt around unused code e.g. ES Lint
and code climate. Upgrade dockerfile to use node v12.

# Pull request checklist
- [ ] Contributed code respects the [editorconfig rules](.editorconfig)
- [ ] Contributed code passes the [eslint rules](.eslintrc.yaml)
- [ ] Contributed code passes the unit tests
- [ ] Added steps are described in the [readme file](README.md)
- [ ] The build of the PR is passing in the Jenkins pipeline(s)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 # Pull request checklist
 - [ ] Contributed code respects the [editorconfig rules](.editorconfig)
-- [ ] Contributed code passes the [eslint rules](.eslintrc.yaml)
 - [ ] Contributed code passes the unit tests
```
Comment (st3v3nhunt) [inline .github/PULL_REQUEST_TEMPLATE.md]: This could be removed now the tests have been removed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
 # Pull request checklist
 - [ ] Contributed code respects the [editorconfig rules](.editorconfig)
```
Comment (st3v3nhunt) [inline .github/PULL_REQUEST_TEMPLATE.md]: I reckon you could get rid of this. If there isn't any automated check for the adherence of the rules it is likely the rule will not be stuck too and be difficult to tell if that is the case for a reviewer.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -3,7 +3,6 @@
...
-- [ ] Contributed code passes the [eslint rules](.eslintrc.yaml)
 - [ ] Contributed code passes the unit tests
 - [ ] Added steps are described in the [readme file](README.md)
```
Comment (st3v3nhunt) [inline .github/PULL_REQUEST_TEMPLATE.md]: The link to the file isn't working so could be updated in this change.

**Review by st3v3nhunt — COMMENTED** [review]
A couple of small points on the only file that looks like it hasn't been updated automatically!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (spattisson) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (spattisson) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed also

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (spattisson) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed anyway

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -2,7 +2,5 @@
 > Please add a brief description of the contents of the pull request and link to the JIRA story
 
 # Pull request checklist
-- [ ] Contributed code respects the [editorconfig rules](.editorconfig)
-- [ ] Contributed code passes the unit tests
-- [ ] Added steps are described in the [readme file](README.md)
-- [ ] The build of the PR is passing in the Jenkins pipeline(s)
+- [ ] Any added cucumber steps are described in the [readme file](README.md)
```
Comment (st3v3nhunt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Any added cucumber steps are described in the [readme file](../README.md)
```

**Review by st3v3nhunt — APPROVED** [review]
LGTM

**Pull request merged**